app/src/main/kotlin/com/mapzen/erasermap/controller/MainActivity.kt

```diff
@@ -1095,6 +1100,18 @@ class MainActivity : AppCompatActivity(), MainViewController, RouteCallback,
         animations.start()
     }
 
+    fun enableSearch() {
+        searchView?.inputType = 1
```

Comment:
Changes look good. The only other thing I see is maybe we should use the constant values `InputType.TYPE_CLASS_TEXT` (1) here and `InputType.TYPE_NULL` (0) below for clarity?